client/components/payment-activity/payment-activity-data.tsx

```diff
@@ -83,6 +84,15 @@ const PaymentActivityData: React.FC = () => {
 					/>
 				}
 				amount={ totalPaymentVolume }
+				handleReportLinkClick={ () => {
+					recordEvent(
+						'wcpay_overview_payment_activity_total_payment_volume_click',
+						{
+							source:
+								'total_payment_volume_tile_view_report_link',
```

Comment:
Do we need the `source` parameter (I'm assuming it's optional)? I think this is used to distinguish generic events by the originating high-level component or container. How do other WooPayments events use `source`? 

In this case we have unique events for each click target, so `source` is redundant. 

An alternative design would be to use a generic event name, and distinguish which box using source:

- en `wcpay_overview_payment_activity_click`
- source `total_payment_volume`

The question is whether we'd want to aggregate any click on the activity widget. If so, a common event is useful. If not, unique event for each thing works well.

Reply:
The idea of using a common event for the tile is fantastic, and makes things totally clean! Thanks for opening my eyes to this possibility! Also a great note for me "Is this a good enough reason to pass a callback?" . Clearly no in this case with the alternative you suggested :D .  

I have implemented a common click event, and a source that is extrapolated from the existing `label` prop. ( make it all small case and replace spaces with underscores, where needed) 

I hope I have done it appropriately here : 5069046

Reply:
Looks good! 

I'd suggest using an explicit `source` value for each tile, rather than generating from label. This is one more prop (`tracksSource`?), but it's simple and explicit, and allows future tracks investigators to search the code for the source value to work out where it's getting fired.

Reply:
I have added an explicit `tracksSource` prop here - 5c5da5b . Thanks!